.travis.yml

```diff
@@ -13,11 +13,17 @@ install:
    - pip install -r requirements/django_requirements.txt
    - pip install -r requirements/elastic_requirements.txt
    - docker pull quen2404/openapi-diff
-script:
-   - py.test --cov=optimade
-   - openapi-spec-validator openapi.json
-     # update schema
-   - python -c "from optimade.server.main import app, update_schema; update_schema(app)"
-     # check whether generated schema matches the committed version
-   - .ci/openapi_diff.sh
-   - codecov --token=$CODECOV_TOKEN
+
+jobs:
+    include:
+        - stage: "Unit Tests"
+          script:
+            - py.test --cov=optimade
+            - codecov --token=$CODECOV_TOKEN
+        - stage: "Schema validation"
+          script:
+            - openapi-spec-validator openapi.json
+            # update schema
+            - python -c "from optimade.server.main import app, update_schema; update_schema(app)"
+            # check whether generated schema matches the committed version
+            - .ci/openapi_diff.sh
```

Comment:
Hmm, was hoping this would give us two travis statuses, one for API and one for tests, but that doesn't seem to be the case.

Reply:
I think you can only do for PR and Push? But at least it creates two different environments on Travis

Reply:
I think I'll just revert this as it doubles build time (I'd also hoped it would only do the install step once but whatever). Going to add pip caches to the CI and leave it there.